x-pack/plugins/security_solution/public/common/components/hover_actions/use_hover_action_items.tsx

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+/* eslint-disable complexity */
```

Comment:
do we want to break this out into its own ticket/tech debt? not sure the long term implications of keeping this in the file, but I know it'd require a bit more in depth refactoring so no worries fixing it in this PR unless you want to

Reply:
Yea, I think it's definitely worth breaking this apart, but will do it in a follow up PR